docs/docs/intro.md

```diff
@@ -1,94 +1,167 @@
 ---
-id: "index"
-title: "👋 Getting Started"
-slug: "/"
+id: 'index'
+title: '👋 Getting Started'
+slug: '/'
 sidebar_position: 0
 custom_edit_url: null
 ---
 
 # GridPlus SDK
 
-The [GridPlus SDK](https://github.com/GridPlus/gridplus-sdk) is designed to facilitate communication between an app or service and a user's [Lattice1 hardware wallet](https://gridplus.io/lattice).
+The [GridPlus SDK](https://github.com/GridPlus/gridplus-sdk) is the bridge between software wallets, like MetaMask or Frame, and the [Lattice1 hardware wallet](https://gridplus.io/lattice).
 
 :::note
 The [Lattice1](https://gridplus.io/lattice) is an Internet-connected device which listens for end-to-end encrypted requests. HTTPS requests originate from this SDK and responses are returned **asynchronously**. Some requests require user authorization and will time out if the user does not approve them.
-
-If you are using the `gridplus-sdk` in a Node.js application with a version of Node lower than v18, you will need to patch the `fetch()` API in the global scope. One solution is to use the `node-fetch` package. See [the `node-fetch` README](https://github.com/node-fetch/node-fetch#installation) for instructions. Other options are available on NPM.
-
 :::
 
-## Installing
+## Getting Started
 
-First install this SDK with:
+### Installation
 
-```bash
+Install the package with your package manager. For example, with npm:
+
+```sh
 npm install --save gridplus-sdk
 ```
 
-## Connecting to a Lattice
+### Connecting to a Lattice
+
+To initiate the connection, you'll call `setup`. This function takes an object with the following properties:
+
+- `name` - the name of the wallet or app you're connecting from, e.g. `MetaMask`
+- `deviceId` - the device ID of the Lattice you're connecting to
+- `password` - an arbitrary string that is used to encrypt/decrypt data for transport
+- `getStoredClient` - a function that returns the stored client data
+- `setStoredClient` - a function that stores the client data
+
+#### Setup Example
+
+`setup()` will return a `boolean` that tells you whether the device has already been paired. If it has not, you will need to pair the device by calling `pair()`.
+
+```ts
+import { setup } from 'gridplus-sdk';
+
+const isPaired = await setup({
+  name: 'My Wallet',
+  deviceId: 'XXXXXX',
+  password: 'password',
+  getStoredClient: () => localStorage.getItem('client'),
```

Comment:
Would be great to also cover async storages since most of the browser extension integrations will require it. If the fix to allow async storages was added to the SDK already, we just probably need to mention it should work fine, if not, then maybe point to the workaround with Zustand I did in Rainbow.